tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -279,6 +279,10 @@ void test_area(const Solution& solution, const Strahlkorper<Fr>& strahlkorper,
       strahlkorper.ylm_spherepack().definite_integral(get(area_element).data());
 
   CHECK_ITERABLE_APPROX(area, expected);
+
+  const double irreducible_mass = StrahlkorperGr::irreducible_mass(area);
+  const double expected_irreducible_mass = sqrt(area / (16.0 * M_PI));
+  CHECK(irreducible_mass == expected_irreducible_mass);
```

Comment:
Since this test simply copies the implementation, so it does not really test anything. We suggest to make `expected_irreducible_mass` another argument to this function. For KerrSchild this should be 
`mass * sqrt(0.5 * (1.0 + sqrt(1.0 - square(magnitude(spin)))))`

Reply:
This is a great suggestion. @jennifersanchez you can just add an argument 

`const double expected_irreducible_mass` to the function `test_area`. Then, right below line 637 of `Test_StrahlkorperGr.cpp` (i.e. the same file), you can add a new variable 
`const double expected_irreducible_mass = mass * sqrt(0.5 * (1.0 + sqrt(1.0 - square(magnitude(spin)))));` and then pass this into `test_area`.